src/main/java/frc/robot/subsystems/Manipulator.java

```diff
@@ -1,41 +1,65 @@
 package frc.robot.subsystems;
 
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj2.command.Commands;
-import edu.wpi.first.wpilibj2.command.SubsystemBase;
-import frc.robot.Constants;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+import edu.wpi.first.wpilibj.motorcontrol.Talon;
+import edu.wpi.first.wpilibj.Solenoid;
+
+import edu.wpi.first.wpilibj.PneumaticsModuleType;
 
 public class Manipulator extends SubsystemBase {
 
-    ManipulatorState state = ManipulatorState.EMPTY;
+    private static Manipulator MANIPULATOR;
 
-    public void init() {}
+    ManipulatorState state = ManipulatorState.UP;
 
-    public ManipulatorState pickUp() {
-        if (state == ManipulatorState.FULL) {
-            return ManipulatorState.ERROR;
-        }
-        //TODO pick up
-        state = ManipulatorState.FULL;
-        return state;
+    Talon coralMotor;
```

Comment:
CoralMotor nebude, bude tam jeden motor na vyhazování Algae, ten se musí akorát otočit o dané otáčky a to stačí